src/pages/Profile.jsx

```diff
@@ -88,13 +88,13 @@ const Profile = () => {
     >
       <Box sx={{ display: 'flex', flexDirection: 'column', alignItems: 'center', gap: '20px' }}>
         <Typography sx={{ fontWeight: 'bold', fontSize: '18px' }}>Profile Information</Typography>
-        <Typography>
-          {client ?? (
+        {!contact ? (
```

Comment:
Initially wanted to keep `??` but I couldn't get it to work. Had to use ternary instead.

Reply:
<img width="857" alt="Screenshot 2023-08-27 at 7 21 22 PM" src="https://github.com/codeforpdx/PASS/assets/71395931/7c590d32-60d0-4024-b450-25df4a5083ee">

This didn't work? This is what Russell has in his PR for deleting notification system.

Reply:
It didn't.

I'm getting a blank page with a rendering error on the console log for some reason. I'm still not sure what's going on.